### PR TITLE
Content for a 5.6.1 update

### PR DESCRIPTION
**Pull request by MadMax3947**

### CODE REVIEW

**Comment by cooljeanius** [comment]
right, *THIS* time I'm going to get #24 done before reviewing...

**Comment by cooljeanius** [comment]
Darn it, looks like I broke the map-diff workflow when trying to set it up for this PR... maybe @macabeus can help? If not, I'll just disable the workflow again...

**Comment by cooljeanius** [comment]
ok so the checks are all green again now at least, time to do the actual review now...

File: dist/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-5.6.0
\ No newline at end of file
+5.6.1
```
Comment (cooljeanius) [inline dist/VERSION:1]: So, is the idea to make a new release after merging this?

File: scenarios/14_Hordes_Undead.cfg [hunk 2]
```diff
@@ -234,19 +234,28 @@
...
+                [message]
+                    id=Kogw
+                    message= _ "I've heard stories of them; step carefully, this is powerful black magic."
+                [/message]
+                [message]
+                    id=Theracar
+                    image=$theracar_image
+                    message= _ "I recall rumors of them from my time serving that wretched Baron; this is powerful black magic."
+                [/message]
+            [/then]
```
Comment (cooljeanius) [inline scenarios/14_Hordes_Undead.cfg:257]: Maybe an `[else]` branch here, like "These statues again?" or something? I dunno...

File: dist/VERSION (hunk 1 above)
Comment (cooljeanius) [reply dist/VERSION:1]: oh wait derp that's the PR's entire name; never mind

File: scenarios/14_Hordes_Undead.cfg (hunk 2 above)
Comment (cooljeanius) [reply scenarios/14_Hordes_Undead.cfg:257]: actually that can wait for a later release; adding such a branch in this PR would give @k-konami extra strings to translate, which probably isn't worth holding up this release for...

**Review by cooljeanius — APPROVED** [review]
I'm going to squash when merging.

File: scenarios/14_Hordes_Undead.cfg (hunk 2 above)
Comment (k-konami) [reply scenarios/14_Hordes_Undead.cfg:257]: Japanese users are used to untranslated material, 
so it's okay if there are some omissions.   :) 

The latest POT(Created on 08-16 19:15 UTC") I checked and there were 37 untranslated.
For the time being, I would be grateful if you could merge the ja.po that is currently Pull-Req'd.

**Pull request merged**